### ISSUE TITLE
Incorrect default location in forms for Rupsa in #30

### ISSUE THREAD

**Issue by mixmixmix** [road86/bahis-desk#35]
### Bug Description

In #30 in form `Configuration->Staff->New Submission` for rupsa is
<img width="564" alt="Screenshot 2022-12-28 at 10 48 51" src="https://user-images.githubusercontent.com/9990867/209793340-39d9bb21-0580-43aa-b299-8efef72a405a.png">

### To Reproduce

1. Log in as `rupsa` 
22 load form `Configuration->Staff->New Submission`

### Expected Behaviour

should be
<img width="600" alt="Screenshot 2022-12-28 at 10 49 04" src="https://user-images.githubusercontent.com/9990867/209793241-7bfcafe4-e2e8-4673-9b85-b8edb39454ac.png">

### Additional Context

This is not a very big problem, can be however related to #34

**Comment by ChasNelson1990** [comment]
So I (now) think this is actually correct - https://www.google.com/maps/place/Rupsha+Upazila,+Bangladesh/@22.8080689,89.5467239,12z/data=!3m1!4b1!4m5!3m4!1s0x39ff9203b1ce5bb1:0x17ceb3869c9a314a!8m2!3d22.8004577!4d89.5868179?hl=en

**Comment by mixmixmix** [comment]
Explain how come?

**Comment by ChasNelson1990** [comment]
I believe this is being caused by the React Warning in the console:

```
Warning: Cannot update a component (`Connect(App)`) while rendering a different component (`XXXX`). To locate the bad setState() call inside `XXXX`, follow the stack trace as described in https://reactjs.org/link/setstate-in-render
```

Where `XXXX` is the different components that are not updating. Why this is a new issue I am not sure but I am looking into it.

**Comment by ChasNelson1990** [comment]
This is actually deep in odkformrenderer and is a repeated antipattern that used to be a silent thing in react but has been a loud warning for a while. Essentially odkformrenderer uses class components but wraps everything inside the render function, including state changes... so it's trying to update central state whilst rendering (bonkers) and React can't allow this because it breaks hooks. So... we either need to rewrite odkformrenderer in proper React... or undo all the new, standard React in the app... or live with it for now?

**Comment by ChasNelson1990** [comment]
This is the relevant issue: https://github.com/facebook/react/issues/18178#issuecomment-595846312
I think this might be a big problem for us...

**Comment by ChasNelson1990** [comment]
This is upstream in odkformrenderer; as I think we need to rewrite odkformrenderer anyway we should discuss this

**Comment by ChasNelson1990** [comment]
This is related to the changes we're making for v2.2.x where we become strict that all users must be an upazila (rupsa's account is a district or division).
Will close with v2.2.x

**Comment by ChasNelson1990** [comment]
Closes by #60
